lib/PlacesFeed.jsm

```diff
@@ -278,12 +278,8 @@ class PlacesFeed {
     }
   }
 
-  fillSearchTopSiteTerm(action) {
-    const gURLBar = action._target.browser.ownerGlobal.document.getElementById("urlbar");
-    gURLBar.value = `${action.data.label} `;
-    gURLBar.focus();
-    // Clear the url bar text selection that happens on focus
-    gURLBar.selectionStart = gURLBar.selectionEnd;
+  fillSearchTopSiteTerm({_target, data}) {
+    _target.browser.ownerGlobal.document.getElementById("urlbar").search(`${data.label} `);
```

Comment:
I see the test file already does getElementById, but this could just be `…ownerGlobal.gURLBar…`